### DISCUSSION TITLE
Minestom community

### DISCUSSION THREAD

**Discussion by DevBlokJochem** [Minestom/Minestom#952] [Help] [answered]
How can I add code to the minestom community server? (And am I allowed todo that?)

**Comment by VendoAU** [comment, marked as answer]
You can make a pull request here:
https://github.com/Minestom/Arena

**Reply by DevBlokJochem** [reply]
And what do I have to do after I did that?

**Reply by TheMode** [reply]
Wait until it gets merged?
